### PR TITLE
Added aristides_2862 controller support for Unicomp keyboards.

### PR DESCRIPTION
**Pull request by purdeaandrei**
## Description

This PR depends on:
EDIT: no more dependencies!
~~#20259 - depends on changes to the outer info.json in this PR~~
~~#20104 - to get rid of lint errors~~

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).
     - only tested that it compiles, and correctly enumerates on USB.

### CODE REVIEW

File: keyboards/unicomp/classic_ultracl_post_2013/aristides_2862/readme.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# unicomp/classic_ultracl_post_2013/aristides_2862
+
+![unicomp/classic_ultracl_post_2013/aristides_2862](https://www.pckeyboard.com/mm5/graphics/00000001/UB4ZPHA_800x343.jpg)
```
Comment (drashna) [inline keyboards/unicomp/classic_ultracl_post_2013/aristides_2862/readme.md:3]: Please rehost these images on Imgur.

Comment (purdeaandrei) [reply keyboards/unicomp/classic_ultracl_post_2013/aristides_2862/readme.md]: I have rehosted some of the images on imgur, but some are linking directly to the manufacturer's website, and they're not available under some open license, so I can't just repost them.

**Comment by purdeaandrei** [comment]
This PR doesn't have any more dependencies (One has been merged, and the other one has been worked around).

File: keyboards/unicomp/spacesaver_m_pre_2013/aristides_2862/rules.mk [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+#
+BOOTMAGIC_ENABLE = yes
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no        # Console for debug
+COMMAND_ENABLE = no        # Commands for debug and configuration
+NKRO_ENABLE = yes           # USB Nkey Rollover
+
+ARISTIDES_COMMON_FILES=../../../unicomp/classic_ultracl_post_2013/aristides_2862
+SRC += $(ARISTIDES_COMMON_FILES)/matrix.c $(ARISTIDES_COMMON_FILES)/aristides_2862.c
```
Comment (drashna) [inline keyboards/unicomp/spacesaver_m_pre_2013/aristides_2862/rules.mk:12]: ideally, these files should be moved to a common folder and included from there

File: keyboards/unicomp/spacesaver_m_post_2013/aristides_2862/aristides_2862.c [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+{
+    // Led pins:
+    // GP4 is the left-most led, normally Num Lock, but on Spacesaver M it's Caps Lock. Configured in info.json
+    writePin(GP3, 1);
+    setPinOutput(GP3); // middle led, always off on Spacesaver M
+
+    writePin(GP2, 1);
+    setPinOutput(GP2); // right-most led, normally Scroll Lock, but on Spacesaver M indicates function layer
+}
+
```
Comment (drashna) [inline keyboards/unicomp/spacesaver_m_post_2013/aristides_2862/aristides_2862.c:29]:
```suggestion
void keyboard_post_init_kb(void) {
    // Led pins:
    // GP4 is the left-most led, normally Num Lock, but on Spacesaver M it's Caps Lock. Configured in info.json
    writePin(GP3, 1);
    setPinOutput(GP3); // middle led, always off on Spacesaver M

    writePin(GP2, 1);
    setPinOutput(GP2); // right-most led, normally Scroll Lock, but on Spacesaver M indicates function layer
    keyboard_post_init_user();
}
```

File: keyboards/unicomp/classic_ultracl_post_2013/aristides_2862/rules.mk
Comment (drashna) [inline keyboards/unicomp/classic_ultracl_post_2013/aristides_2862/rules.mk:1]: most/all of this should be moved to the info.json.  `qmk info -f json` can assist with that.

**Review by drashna — CHANGES_REQUESTED** [review]

**Pull request closed without merging**